src/test/kotlin/it/unibo/protelis/web/execution/simulated/SimulatedProtelisEngineTest.kt

```diff
@@ -0,0 +1,47 @@
+package it.unibo.protelis.web.execution.simulated
+
+import it.unibo.protelis.web.execution.ProtelisObserver
+import it.unibo.protelis.web.execution.ProtelisUpdateMessage
+import kotlin.test.assertTrue
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+
+@DisplayName("Test simulated Protelis engine with Alchemist")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [UndocumentedPublicClass - [SimulatedProtelisEngineTest] at /src/src/test/kotlin/it/unibo/protelis/web/execution/simulated/SimulatedProtelisEngineTest.kt:9:1](https://app.codacy.com/manual/NiccoMlt/Protelis-Web/pullRequest?prid=4969516)